internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -252,6 +274,52 @@ func setIRQLoadBalancing(ctx context.Context, c *oci.Container, enable bool, irq
 	return nil
 }
 
+func setCPUQuota(parentDir string, c *oci.Container) error {
+	var (
+		cgroupManager cgmgr.CgroupManager
+		err           error
+	)
+
+	if strings.HasSuffix(parentDir, ".slice") {
+		if cgroupManager, err = cgmgr.SetCgroupManager("systemd"); err != nil {
+			return nil
+		}
+	} else if cgroupManager, err = cgmgr.SetCgroupManager("cgroupfs"); err != nil {
+		return nil
```

Comment:
wdyt of adding a comment on why we are ignoring the error or logging the error?

applies to line 285 as well

Reply:
good point, I actually updated it to panic because it shouldn't be possible to fail unless there's a programming error

Reply:
lgtm